tests/checker/ftpserver.py

```diff
@@ -50,31 +50,34 @@ def tearDown (self):
 
 
 def start_server (host, port):
-    def line_logger(msg):
+    def line_logger(self, msg):
         if "kill" in msg:
             raise KeyboardInterrupt()
 
     try:
-        from pyftpdlib import ftpserver
+        from pyftpdlib.authorizers import DummyAuthorizer
+        from pyftpdlib.handlers import FTPHandler
+        from pyftpdlib.servers import FTPServer
+        from pyftpdlib import __ver__ as pyftpdlib_version
```

Comment:
I wonder if we want to add a minimum version constraint for `pyftpdlib`?

Reply:
The API changed with Version: 1.0.0 - Date: 2013-02-19
https://github.com/giampaolo/pyftpdlib/blob/master/HISTORY.rst#version-100---date-2013-02-19
(scroll down to Incompatible API changes)

even Debian Jessie had 1.2.0, so we're probably OK?

Reply:
Yeah.